test/core/TilingSprite.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+describe('PIXI.Sprite', function ()
+{
+    describe('getBounds()', function ()
+    {
+        it('must have correct value according to _width and _height', function ()
```

Comment:
The failing test is in the TilingSprite test case, not the Sprite one. It was just incorrectly named, but you can see it comes from the TilingSprite test:

```
/home/travis/build/pixijs/pixi.js/test/core/TilingSprite.js:21:33
```